src/frontend/shortlist/src/App.vue

```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function markSchoolAsTrashed(payload) {
+  console.log(payload);
+  // let apiReq = {
+  //   recoID: payload.recoID,
+  //   trashed: payload.trashed,
+  // };
+  let success = () => {
+    console.log("Added to a list successfully");
+  };
+  let failure = (err) => {
+    console.log("Fail to add to a list.", err.response.data);
+  };
+  let req = apiClient.removeSchoolCard(payload, success, failure);
```

Comment:
I also had a question here. In this case, you seemed to prefer not define the payload above. All the other functions did, was there a particular reason for just passing payload to the apiClient function? @adelaidemz

Reply:
In other functions there is some change that needed to be made to the payload data, which was happening here. In this case the data can be passed exactly as it is emitted elsewhere so it's not necessary

Reply:
Will change the function in shortlist.js after the other one is merged